rainbow-me/rainbowkit
# feat: `ramperWallet`

## Changes
- Added `ramperWallet`

## What to test
1. Install Ramper wallet [here](https://chromewebstore.google.com/detail/ramper-wallet/nbdhibgjnjpnkajaghbffjbkcgljfgdi)
2. Once installed visit [rainbowkit.com](https://www.rainbowkit.com/)
3. Make sure the connecting, disconnecting and auto connecting works smoothly
